BG-1037: admin members

ap-justin opened the pull request:
## Explanation of the solution
* new link in sidebar
<img width="323" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/6b841452-b69b-40fc-964a-cbd0e64d1a25">

* members page
<img width="1251" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/e44606af-a4c0-42b7-b606-c77c8e7254b1">

* clicking `[invite-user]` opens add form. 
<img width="909" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/843865de-e0ce-4c08-a776-b536ad2373b6">

* adding a user immediately puts it on the list regardless whether this user exist or not. Once this user signs-in or creates account. This credential would automatically be added to his `jwt`. Invitee would also receive email

<img width="567" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/7b619be1-adb4-466a-bde3-bf45c87c3da8">

* user can delete other members via delete button
<img width="133" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/7e44fbdb-1fb5-402e-aae8-b4cbaa2edc52">


## Instructions on making this work

- run `yarn` or `yarn install` to install npm dependencies
- run `yarn run test --watchAll` to verify all tests still pass
- (optional) run `yarn run build` to verify the build passes
- run `yarn start` to start the webapp
-

## UI changes for review

When major UI changes are introduced with a PR, please include links to URLS to compare or screenshots demonstrating the difference and notify on design changes

SovereignAndrey commented:
@ap-justin This looks nice at first glance! Will look it over the code changes more. Quick change request based off screenshots though: 
[ ] delete members button should be a more standard icon like "trash bin".

ap-justin commented:
@SovereignAndrey 
added heading and changed delete icon
<img width="1020" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/5fe5374d-96a0-4563-b8d5-c912f55bcb3a">

turns red on active or hover
<img width="1020" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/bf5a12b0-3999-4183-8c8d-8087c31c0076">
